source/jormungandr/jormungandr/instance.py

```diff
@@ -525,6 +525,9 @@ def has_point(self, p):
             return self.geom and self.geom.contains(p)
         except DeadSocketException:
             return False
+        except PredicateError:
```

Comment:
just a little thought: for me, i don't know if catch the exception is a better option than throw the exception(or crash more beautifully..., i know no one like **`server internal error`**)... in this case, catch the exception means, very  probably, a misleading response...

Reply:
But we may find a another coverage than will respond.
This is a first and quick step, I will also update tyr and ed to prevent any invalid shape to be integreted